data/shortcodes.md

```diff
@@ -16,6 +16,7 @@
 | 0101      | parole-religieuse    | Embarras de la parole religieuse                                                                                                       |               |       |
 | 0102      | scenario-tanner      | Le scénario chez Alain Tanner                                                                                                          |               |       |
 | 0103      | theatre-societe      | Theatre de societe                                                                                                                     |               |       |
+| 0104      | [free to use]        |
```

Comment:
Here I'm not 100% sure if this is free. https://app.ls-prod-server.dasch.swiss/project/0104 gives me an empty page, so I guess such a project doesn't exist. If you want to be 100% sure, you could revert this line.